extension/src/popup/views/AccountHistory/index.tsx

```diff
@@ -30,12 +30,16 @@ const getIsPayment = (type: Horizon.OperationResponseType) =>
     Horizon.OperationResponseType.pathPaymentStrictSend,
   ].includes(type);
 
+const getIsSwap = (operation: HistoryItemOperation) =>
+  operation.type_i === 13 && operation.source_account === operation.to;
+
+enum SELECTOR_OPTIONS {
+  ALL = "ALL",
+  SENT = "SENT",
+  RECEIVED = "RECEIVED",
+}
```

Comment:
no need to change this, but wanted to point out why i had moved `SELECTOR_OPTIONS` inside this component:

I needed translations for `SELECTOR_OPTIONS`, but the job that builds translations can't interpret code, it can only analyze it statically for strings inside of `t()`. So, a common workaround is to include a comment where you manually "call" the `t()` function so the builder knows to translate those strings:
``  
/*
      t("ALL");
      t("SENT");
      t("RECEIVED");
    */`
```

This comment needs to be inside of a React component for the builder to analyze it. I had moved the enum inside to be next to the comment so it was clear that they were related

Reply:
ahh gotchya makes sense 👍